### DISCUSSION TITLE
易用性不太适合小白用户

### DISCUSSION THREAD

**Discussion by lopins** [1Panel-dev/1Panel#5544] [General]
PHP拓展对小白用户来说完全一头雾水，建议完善常见开源CMS的拓展配置方案，目前只有WordPress和少数几个CMS，并且还不全，比如Pg拓展和sqlite拓展都没在常见选项中

**Comment by wanghe-fit2cloud** [comment]
麻烦将您需要的常用项目、扩展等，整理到 github issue 中，我们后续版本适配。

**Comment by bddjr** [comment]
标题是不是应该改成 `易用性不太适合PHP用户`
